components/BottomNavigation/src/MDCBottomNavigationBar.h

```diff
@@ -14,6 +14,7 @@
 
 #import <UIKit/UIKit.h>
 
+#include "MDCAvailability.h"
```

Comment:
This exposes a hopefully private header.

Maybe we should just use
`defined(__IPHONE_13_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0)`

However I do like the readability of the current macro.

Reply:
It is a nice macro, but (as I posted elsewhere), it should be added independently of this change.